shell/common/platform_view.cc

```diff
@@ -61,6 +61,8 @@ void PlatformView::SetViewportMetrics(const ViewportMetrics& metrics) {
   delegate_.OnPlatformViewSetViewportMetrics(metrics);
 }
 
+void PlatformView::SetPixelGeometry(SkPixelGeometry pixel_geometry) {}
```

Comment:
Missing implementation. I realize you are splitting the embedder specific bits into its own patch but the embedder agnostic bits should be complete.

Reply:
The embedder platform view is the only one that actually does anything with this. It's a no-op on other platforms - I'm just leaving this implementation as empty so that we aren't forced to implement no-ops all over the place.

In the embedder platform view, it passes this on to the surface directly.